bgpd/bgp_route.c

```diff
@@ -11885,6 +11885,12 @@ static void show_adj_route(struct vty *vty, struct peer *peer, afi_t afi,
 
 		vty_out(vty, "%s\n", json_object_to_json_string_ext(
 					     json, JSON_C_TO_STRING_PRETTY));
+
+		if (output_count == 0 && filtered_count == 0) {
```

Comment:
I would prefer using shortened version `if (!output_count && !filtered_count)`

Reply:
Thanks, @ton31337. I replaced this check with the shortened version, as suggested.
Kindly have a look.